src/checklyhq-provider.ts

```diff
@@ -0,0 +1,131 @@
+/* Copyright © 2022 Seneca Project Contributors, MIT License. */
+import { ChecklyhqProviderOptions } from './types'
+import { getJSON, makeConfig, makeUrl, postJSON } from './utils'
+
+const Pkg = require('../package.json')
+
+import fetch from "node-fetch";
+
+function CheklyhqProvider(this: any, options: ChecklyhqProviderOptions) {
+  const seneca: any = this
+
+  const entityBuilder = this.export('provider/entityBuilder')
+
+  seneca.message('sys:provider,provider:checklyhq,get:info', get_info)
+
+  async function get_info(this: any, _msg: any) {
+    return {
+      ok: true,
+      name: 'checklyhq',
+      version: Pkg.version,
+    }
```

Comment:
let's add a `mark` property here that comes from the msg - this allows you to debug by making sure you get back what you send

Reply:
Ok, like this?
    
    return {
      ok: true,
      name: 'checklyhq',
      version: Pkg.version,
      mark: 'checklyhq-provider',
    }